packages/node_modules/@webex/internal-plugin-ediscovery/src/ediscovery.js

```diff
@@ -43,7 +42,24 @@ const EDiscovery = WebexPlugin.extend({
       service: 'ediscovery',
       resource: 'reports',
       body
-    });
+    })
+      .catch((reason) => {
+        // This value correspondes to the ediscovery error code INVALID_USER_EMAILS_LIST_IN_MESSAGE
+        if (reason.body.errorCode === 14400001) {
+          try {
+            const invalidEmails = JSON.parse(reason.body.message);
+
+            if (Array.isArray(invalidEmails) && invalidEmails.length) {
+              return Promise.reject(new InvalidEmailAddressError(invalidEmails));
+            }
+          }
+          catch (error) {
+            // assume syntax error and continue
+          }
+        }
+
+        return Promise.reject(reason);
+      });
```

Comment:
We currently supply a pre-existing custom error to the Atlas UI if their request contained invalid emails, this is still the case and requires no change on their part